fdbrpc/include/fdbrpc/ReplicationPolicy.h

```diff
@@ -156,6 +156,7 @@ struct PolicyAcross final : IReplicationPolicy {
 	const std::string& attributeKey() const { return _attribKey; }
 
 protected:
+	Optional<bool> _cacheable;
```

Comment:
why is this optional? it seems you only check if this is present or not, thus a simple `bool` is good enough.